src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/entity/Adjustment.kt

```diff
@@ -92,4 +93,9 @@ data class Adjustment(
     taggedBail?.adjustment = this
     additionalDaysAwarded?.adjustment = this
   }
+
+  fun getBookingIdFromLegacyData(): String? {
```

Comment:
Have a look at how the objectMapper is used within `LegacyService` to get the booking ID. Just to keep the code consistent, more type safe and also uses the same objectmapper as the rest of the service.

Reply:
Code amended.